.pfnci/script.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# script.sh is a script to run Docker for testing.  This is called by CI like
+# "bash .pfnci/script.sh py37".  This script should also be designed to be
+# called in a local machine.  If a local machine has no GPUs, this should fall
+# back to CPU testing automatically.
```

Comment:
It seems this script is assumed to be run on Google Cloud. How about mentioning that here?

Reply:
Done. Plus, I added a TODO to enable external contributors to reproduce it. This requires another PR on CuPy.